### PR TITLE
Adding all iterm2 colorschemes + ruby converter

### PR DESCRIPTION
**Pull request by metalelf0**
I had some spare time this weekend so I put together a small ruby script to convert all the color palettes from prot16 to iterm2 colorschemes.

I generated both light and dark variants; you can see how I mapped prot16 labels to iterm2 values under `LIGHT_PROFILE_MAPPINGS` and `DARK_PROFILE_MAPPINGS` in `utils/prot16_to_iterm2.rb`. The color conversion is done in the `Color` class.

I have tested only a couple of them and they look good, if anything looks odd let me know and I'll do my best to fix it :)

### CODE REVIEW

File: utils/prot16_to_iterm2.rb [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>'.freeze
+
+FILE_FOOTER = '</dict>
+</plist>'.freeze
+
+scheme_names = %w(alto archaic bionis blau caprice cyprium equinox ficus flowerbed gaia magus nefelio neptune ocarina oliveira orionis overgrowth playa seabed sonho symbiosis vin)
+scheme_names.each do |scheme_name|
+  input_file = "#{scheme_name}/lb/#{scheme_name}-values.yml"
```
Comment (protesilaos) [inline utils/prot16_to_iterm2.rb]: I was in the process of cleaning up the repo. I thus changed this to #{scheme_name}/general/#{schemme_name}-values.yml

**Review by protesilaos — CHANGES_REQUESTED** [review]

**Comment by protesilaos** [comment]
This is awesome work! Many thanks! 

It also comes at a time when I was reviewing the entire repository (and added the Terminator ports/configs).

All looks good to me. I just had a minor note where I requested a change. 

Also, as you can see from the `terminator` dir under each item, I am including a README ([example](https://github.com/protesilaos/prot16/tree/master/alto/terminator)) with a proviso:

> This is experimental. Not all use cases have been checked. Colour mapping may require further refinements. Also note that I have only tested this on my Arch Linux and Linux Mint machines.

I can write something similar for all your contributions. What do you think?

**Comment by metalelf0** [comment]
@protesilaos very good! I updated the path as you suggested. :+1: for the proviso `README`; I tested some of the schemes with iTerm2 3.0 daily builds under macOS Sierra 10.12.2 and they look consistent.

**Comment by protesilaos** [comment]
Excellent! Much appreciated. 

Will add the iterm2 `README` shortly. 

Also, I will update the ["thanks to" section](https://github.com/protesilaos/prot16#thanks-to) to provide full attribution to your work. Same with my website's scheme pages. I will add a section about iterm2, plus reference to your work.

**Pull request merged**